Item/Item.module.less

```diff
@@ -156,37 +210,45 @@
 			}
 		}
 
+		.disabled({
+			color: @agate-item-disabled-color;
+			border-color: @agate-item-disabled-border-color;
+			opacity: @agate-item-disabled-opacity;
+		});
```

Comment:
Would you please describe why did you add this?

Reply:
I needed to overwrite the styles for `disabled` which comes from `ThemeDecorator`, in order for the Item to have the same opacity when `disabled`. 
GUI did not take into consideration the @agate-disabled-opacity: 0.6 which is applied in ThemeDecorator

Reply:
🤔  I think it's okay to leave as it is.

Reply:
👍